packages/components/organisms/f-status-banner/src/services/StatusBannerServiceApi.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+export default {
+    async dummyService (url, tenant, data) {
```

Comment:
Is this a placeholder for future code?

Reply:
Similar to above. Left in incase I need it. Will delete in the next PR if not.